tests/lib-bench/src/miniboxing/benchmarks/rrbvector/miniboxed/MbLLSRegression.scala

```diff
@@ -0,0 +1,48 @@
+package miniboxing.benchmarks.rrbvector.miniboxed
+
+import miniboxing.runtime.math.MiniboxedFractional
+
+// Learn Least Square Regression using RRBVector
+class MbLLSRegression[@miniboxed A](val x: RRBVector[A], val y: RRBVector[A])(implicit numeric: MiniboxedFractional[A]) {
+  def sum_x: A = x.sum
+  def sum_y: A = y.sum
+  def sum_xy: A = {
+    var sum = numeric.zero
+    var i = 0
+    while (i < x.length) {
+      sum = numeric.plus(sum, numeric.times(x(i), y(i)))
+      i = i + 1
+    }
+    sum
+  }
```

Comment:
@milosstojanovic, just thinking aloud: I suspect that doing:
```
def sum_xy: A = x zip y map { case (x, y) => x * y } sum
```
would be more idiomatic and would provide a better speedup. Can you try it out?